.docksal/docksal.yml

```diff
@@ -1 +1,19 @@
 version: "2.1"
+services:
+  # Solr
+  solr:
+    hostname: solr
+    image: docksal/solr:8
```

Comment:
Solr 8 won't do. It will have to be 7.7.3.

Reply:
I'll test locally, but should be fine

Reply:
Done, config updated